src/keys/KeyManager.ts

```diff
@@ -419,11 +472,12 @@ class KeyManager {
     duration: number = 31536000,
     subjectAttrsExtra: Array<{ name: string; value: string }> = [],
     issuerAttrsExtra: Array<{ name: string; value: string }> = [],
-  ): Promise<void> {
+  ): Promise<string> {
     this.logger.info('Resetting root key pair');
     const keysDbKeyPlain = await this.readKey(this.dbKeyPath);
     const keysVaultKeyPlain = await this.readKey(this.vaultKeyPath);
-    const rootKeyPair = await this.generateKeyPair(bits);
+    const mnemonic = keysUtils.generateMnemonic();
```

Comment:
Replace all uses of `mnemonic` with `recoveryCode` instead.

Reply:
Fixed.